openedx/features/pakx/lms/pakx_admin_app/permissions.py

```diff
@@ -2,12 +2,15 @@
 Permissions for PakX Admin Panel APIs.
 """
 from rest_framework.permissions import BasePermission
+from django.contrib.auth.models import User
 
-from .constants import GROUP_TRAINING_MANAGERS
+from .constants import GROUP_TRAINING_MANAGERS, GROUP_ORGANIZATION_ADMIN
 
 
 class CanAccessPakXAdminPanel(BasePermission):
     def has_permission(self, request, view):
-        return request.user.groups.filter(
-            name=GROUP_TRAINING_MANAGERS
-        ).exists() or request.user.is_superuser
+        return request.user.is_superuser or User.objects.filter(
+            groups__name__in=[GROUP_TRAINING_MANAGERS, GROUP_ORGANIZATION_ADMIN],
+            profile__organization__isnull=False,
+            id=request.user.id
```

Comment:
will it matter if we filter users on id first before filtering on other complex fields?